client/my-sites/stats/post-detail-table-section/style.scss

```diff
@@ -0,0 +1,4 @@
+.segmented-control.stats-views__table-control {
+	max-width: 280px;
```

Comment:
```suggestion
	max-width: fit-content;
```

With `280px`, the buttons aren't quite centered:

<img width="1065" alt="image" src="https://user-images.githubusercontent.com/4044428/207455086-aa2c53da-4881-40d2-8563-d0e54b3bdbf7.png">

With [fit-content](https://caniuse.com/?search=fit-content), the buttons are correctly centered:

<img width="1074" alt="image" src="https://user-images.githubusercontent.com/4044428/207455391-bafa88cb-4c23-431a-a43b-660674349477.png">